app/map/page.tsx

```diff
@@ -150,11 +150,11 @@ export default function App() {
 
   const layers = [
     new PMTLayer({
-      id: "regions-0-10-zoom",
+      id: "regiontile",
       data: REGION_URL,
       onHover: handleTooltipInfo,
       minZoom: 2,
-      maxZoom: 10,
+      maxZoom: 11,
```

Comment:
@nofurtherinformation will this change prevent the block layer from rendering at zoom 10? Made this change b/c right now region goes down to 10 and block starts at 11 creating a gap.

Reply:
Not quite -- this sets the highest zoom to query from the dataset. I just made this a bit more explicit in the `TRANSITION_ZOOM_THRESHOLD` variable that controls the rendering now in map>page